europe_nuclear_paper.tex

```diff
@@ -10,9 +10,10 @@
 \usepackage[hidelinks]{hyperref}
 \usepackage{xspace}
 \usepackage{hhline}
+\usepackage{amsmath}
 
 \usepackage{tabularx}
-\newcolumntype{b}{X}
+\newcolumntype{b}{>{\hsize=1.0\hsize}X}
```

Comment:
This seems pointless.

Reply:
I left this like this just in case I need to edit the table widths again, and then I can expand it or reduce it!

Reply:
up to you. I would just use X in place in the table and not use a newcolumntype declaration at all. But that's me.